ansible/group_vars/all.yml

```diff
@@ -23,6 +23,6 @@ rocketmq_name_server_port: 9876
 rocketmq_broker_port_main: 10911
 
 storage_fs_type: ext4
-storage_device: /dev/nvme1n1
+storage_device: /dev/sdf
```

Comment:
It seems there was some change on aws side.
EBS volumes are exposed as NVMe block devices on instances built on the Nitro System. 
T2 is not listed anymore as build on Nitro System:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances

Reply:
Could not find AWS announcement on this, but they must have stopped supporting NVMe for T2 small, per the docs mentioned.